index.html

```diff
@@ -0,0 +1,313 @@
+} else if (condition_n) {
+  statement_n;
+} else {
+  statement_last;
+}</code>
+      </p>
+      <p>In the case of multiple conditions only the first logical condition which evaluates to true will be executed. To execute multiple statements, group them within a block statement ({ ... }) . In general, it's good practice to always use block statements, especially when nesting if statements:<code>if (condition) {
+  statement_1_runs_if_condition_is_true;
+  statement_2_runs_if_condition_is_true;
+} else {
+  statement_3_runs_if_condition_is_false;
+  statement_4_runs_if_condition_is_false;
+}</code>
+      </p>
+      <p>It is advisable to not use simple assignments in a conditional expression, because the assignment can be confused with equality when glancing over the code. For example, do not use the following code:<code>if (x = y) {
+  /* statements here */
+}</code>
+      </p>
+      <p>If you need to use an assignment in a conditional expression, a common practice is to put additional parentheses around the assignment. For example:<code>if ((x = y)) {
+  /* statements here */
+}</code>
+      </p>
+      </article>
+    </section>
+
+    <section id="while_statement" class="main-section">
+      <header>
+        <h2>while statement</h2>
+      </header>
+      <article>
+      <p>A while statement executes its statements as long as a specified condition evaluates to true. A while statement looks as follows:<code>while (condition)
+  statement</code>
+      </p>
+      <p>If the condition becomes false, statement within the loop stops executing and control passes to the statement following the loop.</p>
+      <p>The condition test occurs before statement in the loop is executed. If the condition returns true, statement is executed and the condition is tested again. If the condition returns false, execution stops and control is passed to the statement following while.</p>
+      <p>To execute multiple statements, use a block statement ({ ... }) to group those statements.</p>
+      <p>Example:</p>
+      <p>The following while loop iterates as long as n is less than three:<code>var n = 0;
+var x = 0;
+while (n &lt 3) {
+  n++;
+  x += n;
+}</code>
+      </p>
+      <p>With each iteration, the loop increments n and adds that value to x. Therefore, x and n take on the following values:
+        <ol>
+          <li>After the first pass: n = 1 and x = 1</li>
+          <li>After the second pass: n = 2 and x = 3</li>
+          <li>After the third pass: n = 3 and x = 6</li>
+        </ol>
+      </p>
+      <p>After completing the third pass, the condition n &lt 3 is no longer true, so the loop terminates.</p>
+      </article>
+    </section>
+
+    <section id="Function_declarations" class="main-section">
+      <header>
+        <h2>Function declarations</h2>
+      </header>
+      <article>
+      <p>A function definition (also called a function declaration, or function statement) consists of the function keyword, followed by:
```

Comment:
As mentioned in other comments, it is important to respect a proper indentation structure to make the code more readable. Like in a text, it will have several paragraphs, and it's own styling.

Reply:
Hi, @cancelei!! 👋
- Well done! Here you have made a good suggestion with reasoning. 👍 

- [ ] Kindly phrase the comments politely as suggestions instead of orders.
  - For example:
    - After the suggestion you can add this: "So, kindly fix the indentation following the HTML linters setup.......". 
- [ ] Kindly use emoticon (😄) so that your review looks human-friendly.